import_data_nianiapl.ipynb

```diff
@@ -0,0 +1,443 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "65e4fae4",
+   "metadata": {},
+   "source": [
+    "Below you can find the code to import data from niania.pl website"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "64eb9c8e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import requests\n",
+    "from bs4 import BeautifulSoup\n",
+    "import pandas as pd"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "12637b9b",
+   "metadata": {},
+   "source": [
+    "how many pages nianiapl has for chosen location:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "bb21d2d7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pages = 8"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "72a8f134",
+   "metadata": {},
+   "source": [
+    "create set to store offer links:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "b5f80b64",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "links = set()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "34a28cf9",
+   "metadata": {},
+   "source": [
+    "create dictionary from which we will create df later:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "7ca516b3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dict = {'date': [], 'address': [], 'salary': [], 'employment': [], 'author': [], 'experience': [],\n",
```

Comment:
nie powinno się używać słów z pythona jako nazwy zmiennych bo potem są problemy z wykonywaniem kodu jak chcesz użyć np. dict. Możesz to nazwać jako` features` albo jak koniecznie chcesz podkreślić, ze to słownik to jako `features_dict`